.github/workflows/pre-deploy.yaml

```diff
@@ -36,8 +36,8 @@ jobs:
     - name: Run TestCafe Tests
       run: |
         chmod +x "${GITHUB_WORKSPACE}/scripts/clearBrowserStackJobs.sh"
-        "${GITHUB_WORKSPACE}/scripts/clearBrowserStackJobs.sh"
         npm run test:functional:ci:predeploy
+        "${GITHUB_WORKSPACE}/scripts/clearBrowserStackJobs.sh"
```

Comment:
I'm curious to know why you switched this to be executed after running the tests instead of before. If tests fail, will it still run the `clearBrowserStackJobs` script?